music_assistant/__main__.py

```diff
@@ -34,19 +34,6 @@
 def get_arguments() -> argparse.Namespace:
     """Arguments handling."""
     parser = argparse.ArgumentParser(description="MusicAssistant")
-
-    default_data_dir = os.getenv("APPDATA") if os.name == "nt" else os.path.expanduser("~")
-    if not default_data_dir:
-        parser.error("Unable to find default data dir")
-    default_data_dir = os.path.join(default_data_dir, ".musicassistant")
-
-    parser.add_argument(
-        "-c",
-        "--config",
-        metavar="path_to_config_dir",
-        default=default_data_dir,
-        help="Directory that contains the MusicAssistant configuration",
-    )
```

Comment:
the commandline argument must be kept. Just add a second one for the cache dir.
Just set the default to the XDG_DATA_HOME env var.

I think the nicest way to handle all this is by specifying a data-dir and cache-dir argument and add config as alias (for backwards compatibility).